src/main/java/github/weichware10/util/Files.java

```diff
@@ -10,6 +12,9 @@
 import java.net.URL;
 import java.util.Arrays;
 import java.util.List;
+
+import javax.imageio.ImageIO;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'javax.imageio.ImageIO'